### PR TITLE
dropboxd_status: use helpers, deprecate stuffs, fix exception, add strings, bugfix. 

### PR DESCRIPTION
**Pull request by lasers**
Adds `self.py3.command_output()`
Deprecates `{}`
Adds placeholder `{status}`
Add strings
* string_busy: show when Dropbox busy (default None)
* string_off: show when Dropbox unavailable (default None)
* string_on: show when Dropbox up-to-date (default None)

Adds something to take care of exception. Closes https://github.com/ultrabug/py3status/issues/717

* string_unavailable: show when Dropbox isn't installed (default 'Dropbox isn't installed!')

### CODE REVIEW

File: py3status/modules/dropboxd_status.py [hunk 1]
```diff
@@ -3,55 +3,87 @@
 Display status of Dropbox daemon.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 10)
-    format: prefix text for the dropbox status (default 'Dropbox: {}')
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default "Dropbox: {status}")
+    status_busy: show when Dropbox is busy (default None)
+    status_off: show when Dropbox isn't running (default "isn't running")
+    status_on: show when Dropbox is up to date (default "Up to date")
```
Comment (tobes) [inline py3status/modules/dropboxd_status.py]: the status_.. descriptions should be more like

`text for placeholder {status} when up-to-date`

File: py3status/modules/dropboxd_status.py [hunk 2]
```diff
@@ -3,55 +3,87 @@
...
 
-Valid status values include:
+Value for `status_on` if left None:
     - Dropbox isn't running!
+Value for `status_off` if left None:
+    - Up to date
+Values for `status_busy` if left None:
+    - Connecting...
```
Comment (tobes) [inline py3status/modules/dropboxd_status.py]: I don't really like this whole section. `Value for `status_on` if left None:` reads more as code than as documentation.

Not sure how to do this best.

File: py3status/modules/dropboxd_status.py [hunk 3]
```diff
@@ -3,55 +3,87 @@
...
+    color_good: Up to date
 
 Requires:
-    dropbox-cli: command line tool
+    dropbox-cli: command line interface for dropbox
 
 @author Tjaart van der Walt (github:tjaartvdwalt)
 @license BSD
 """
-
```
Comment (tobes) [inline py3status/modules/dropboxd_status.py]: these lines should not be deleted. see #740 

You are getting the hang of stuff so I'm going to start expecting higher quality PRs

Comment (lasers) [inline py3status/modules/dropboxd_status.py]: Noted. It came back.

File: py3status/modules/dropboxd_status.py (hunk 2 above)
Comment (lasers) [inline py3status/modules/dropboxd_status.py]: I changed slightly to `if not set` to make it less code-ish and did `text for placeholder {status}...` bit. I think I like the old one `shows when Dropbox...` (much shorter).

**Pull request merged**